Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -283,13 +294,20 @@ public void SaveCurrentState()
             if (grabbingObject == null)
             {
                 previousParent = transform.parent;
-                previousKinematicState = rb.isKinematic;
+
+                if (rb)
```

Comment:
climbable (handhold) objects don't need/create a rigid body since their could be a ton of them and I didn't want to slow down the physics system. Because of that these could potentially be null and I think need to be checked.